Cargo.toml

```diff
@@ -17,7 +17,7 @@ resolver = "2"
 repository = "https://github.com/lalrpop/lalrpop"
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
 license = "Apache-2.0 OR MIT"
-version = "0.22.0" # LALRPOP
+version = "0.22.1" # LALRPOP"
```

Comment:
Nit: seems an extra `"` showed up at the end here.

Reply:
Fixed both this and the version.sh bug that introduced the extra `"`